tests/test_symbolic_coefficients.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+import pytest
+
+from conftest import skipif
+from devito import Grid, Function, TimeFunction, Eq, Coefficient, Substitutions
+from devito.finite_differences import Differentiable
+
+_PRECISION = 9
+
+pytestmark = skipif(['yask', 'ops'])
+
+
+class TestSC(object):
+    """
+    Class for testing symbolic FD coefficients
```

Comment:
nitpicking: drop "FD"
nitpicking: no need to break over two lines

Reply:
Changed.